canvas_modules/harness/package.json

```diff
@@ -4,15 +4,15 @@
   "private": true,
   "license": "Apache-2.0",
   "scripts": {
-    "start": "NODE_ENV=development node index.js",
-    "build": "npx stylelint 'src/**/*.scss' && grunt",
+    "start": "export NODE_OPTIONS=--openssl-legacy-provider && NODE_ENV=development node index.js",
```

Comment:
@nmgokhale will this be fixed with your carbon 11 update and updating webpack.

Reply:
In my carbon 11 PR, these changes work. Didn't add `export NODE_OPTIONS=--openssl-legacy-provider &&` in any of the files. Test harness app runs OK. I didn't test all the components though -
<img width="869" alt="Screenshot 2024-03-11 at 10 13 25 AM" src="https://github.com/elyra-ai/canvas/assets/25124000/5ceba213-d954-4593-9062-c64053b50530">

<img width="1728" alt="Screenshot 2024-03-11 at 10 13 50 AM" src="https://github.com/elyra-ai/canvas/assets/25124000/fbf34b6f-db53-4c88-8f97-370b29c2f2ce">